bin/triton-tensor-layout.cpp

```diff
@@ -180,6 +181,7 @@ int main(int argc, char **argv) {
   DialectRegistry registry;
   // Register all dialects that can print tensor layout.
   registry.insert<triton::gpu::TritonGPUDialect>();
+  registry.insert<triton::nvidia_gpu::TritonNvidiaGPUDialect>();
```

Comment:
should we call `registerTritonDialects` so that we covert any kind of ops?

Reply:
That seems totally reasonable, yes!

Reply:
are planning to update the PR or do you prefer landing it that way and someone can generalize it later?

Reply:
Yes, planning to update -- will do it this afternoon!